src/Commands/AverageDistance.cpp

```diff
@@ -0,0 +1,47 @@
+#include "AverageDistance.h"
+
+double left_measurement;
+double right_measurement;
+
+AverageDistance::AverageDistance()
+{
+	// Use Requires() here to declare subsystem dependencies
+	// eg. Requires(chassis);
+	Requires(ultrasonic612);
+}
+
+// Called just before this Command runs the first time
+void AverageDistance::Initialize()
+{
+	left_measurement = ultrasonic612->measureL();
+	right_measurement = ultrasonic612->measureR();
+	//AverageDistance::isDone = false;
```

Comment:
why measure left and right? is it just for the ping to make sure they're on same page?